### ISSUE TITLE
Return Sequence Instead of logits

### ISSUE THREAD

**Issue by whitead** [ur-whitelab/wazy#7]

**Comment by whitead** [comment]
Won't fix - a bit complicated to implement and unclear benefit
